doc/classes/Window.xml

```diff
@@ -685,7 +685,10 @@
 			If [code]true[/code], the [Window] can't be focused nor interacted with. It can still be visible.
 		</member>
 		<member name="unresizable" type="bool" setter="set_flag" getter="get_flag" default="false">
-			If [code]true[/code], the window can't be resized. Minimize and maximize buttons are disabled.
+			If [code]true[/code], the window can't be resized. Maximize button is disabled.
```

Comment:
If `unresizable` is no longer enough to stop minimizing this breaks compatibility